rskj-core/src/main/java/co/rsk/peg/BridgeSupport.java

```diff
@@ -1334,6 +1334,30 @@ public int getBtcBlockchainInitialBlockHeight() throws IOException {
         return blockLocator;
     }
 
+    public byte[] getBtcBlockHeaderByHash(Sha256Hash hash) throws IOException, BlockStoreException {
+        this.ensureBtcBlockStore();
+
+        return serializeBlockHeader(btcBlockStore.get(hash));
+    }
+
+    public byte[] getBtcParentBlockHeaderByHash(Sha256Hash hash) throws IOException, BlockStoreException {
+        this.ensureBtcBlockStore();
+
+        StoredBlock block = btcBlockStore.get(hash);
+
+        if (block == null) {
+            return ByteUtil.EMPTY_BYTE_ARRAY;
+        }
+
+        return serializeBlockHeader(btcBlockStore.get(block.getHeader().getPrevBlockHash()));
+    }
+
+    public byte[] getBtcBlockchainBestBlockHeader() throws BlockStoreException, IOException {
+        this.ensureBtcBlockStore();
+
+        return serializeBlockHeader(btcBlockStore.getChainHead());
```

Comment:
```suggestion
        return serializeBlockHeader(getBtcBlockchainChainHead());
```
Just a suggestion, it doesn't really change much. It just reuses that method we already have

Reply:
Good catch, done!